### DISCUSSION TITLE
Screen size and resolution

### DISCUSSION THREAD

**Discussion by waywardmonkeys** [Interlisp/medley#154] [Q&A]
I've been doing a lot of work on Maiko, so I thought it might be cool to actually look at the Medley side of things and try running all of this stuff rather than just working on the VM only.

I'm using a 4k display though and the window size is small and I can't read anything. Short of getting glasses or changing my monitor resolution or getting another monitor to run things ... do I have other good options?

**Comment by masinter** [comment]
https://user-images.githubusercontent.com/1116587/104874843-bb2f9400-5908-11eb-845b-750af614b164.MOV

I was pretty sure the display was running 4K and the X-server was pixel doubling

**Reply by waywardmonkeys** [reply]
How did you do that?

**Comment by masinter** [comment]
https://www.raspberrypi.org/forums/viewtopic.php?t=204150

It was set up by default.

**Reply by waywardmonkeys** [reply]
I got something partly working via `xpra` and a `run_scaled` script, but that doesn't have fully working input. Something goes wrong with the shift key.

**Reply by masinter** [reply]
note there are problems with shift-lock. Some keyboards have shift-lock state and others rely on the system. I changed the lisp default to set the keyaction of the lock key to toggle shift-lock status. You might try the Xvnc route also. You didn't say what OS / desktop you were running,

**Reply by waywardmonkeys** [reply]
I'm thinking about using Xkb within the X11 code in Maiko and seeing if that helps with some input issues. But that's a separate thing.

I'm running Ubuntu 20.04 on an Intel NUC with a 24" LG display at 4k.

**Comment by waywardmonkeys** [comment]
I've decided to just change monitor resolution for now when I want to run `lde` until I can just go and improve how `lde` handles scaling.
